### PR TITLE
feat: enterpriseStart command to create or start a simulation

### PR DESCRIPTION
**Pull request by tpetillot**
**Motivation:**
We wan't a single command to create / start simulation, interactively or not.
**Modifications:**
* If simulationId is configured, upload to simulation package and start the simulation
* If not, and batch mode option is configured, try to create with default values a package and a simulation using it, then upload to package and start simulation
* Otherwise, start interactive creation or start of a simulation

**Ref:** MISC-318

### CODE REVIEW

File: src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala [hunk 1]
```diff
@@ -77,97 +86,180 @@ object EnterpriseSettings {
...
     val file = buildEnterprisePackage(config).value
     val settingPackageId = (config / enterprisePackageId).value
-    val client = httpEnterpriseClient(config).value
+    val enterprisePlugin = enterpriseClientTask(config).value
+    val logger = streams.value.log
 
     if (settingPackageId.isEmpty) {
       throw new IllegalStateException(
-        s"${config.id} / enterprisePackageId has not been specified. See https://gatling.io/docs/enterprise/cloud/reference/user/package_conf/ to create one."
+        s"""A package ID is required to upload a package on Gatling Enterprise; see https://gatling.io/docs/enterprise/cloud/reference/user/package_conf/, create a package and copy its ID.
```
Comment (ccedric) [inline src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala:150]: Missing upload with simulation id feature

File: src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala [hunk 2]
```diff
@@ -77,97 +86,180 @@ object EnterpriseSettings {
...
-           |${config.id} / enterpriseSimulationId := "${simulation.id}"
-           |You may also want to only upload your packaged simulation by using 'enterpriseUpload' command with 'enterprisePackageId' configured:
-           |${config.id} / enterprisePackageId := "${simulation.pkgId}"
-           |""".stripMargin
+    ).recoverWith { case e: SeveralTeamsFoundException =>
+      val teams = e.getAvailableTeams.asScala
+      Failure(
+        new IllegalArgumentException(
+          s"""
+             |More than 1 team were found to create a simulation.
```
Comment (ccedric) [inline src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala:229]: while creating a simulation

File: src/main/scala/io/gatling/sbt/settings/BaseSettings.scala [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+import _root_.io.gatling.sbt.GatlingPlugin.gatlingTestFramework
+
+import sbt._
+import sbt.Keys._
+import sbt.Tests.{ Argument, Group }
+
+object BaseSettings {
+
+  private def overrideDefaultJavaOptions(javaOptions: String*): Seq[String] =
```
Comment (andrewregas) [inline src/main/scala/io/gatling/sbt/settings/BaseSettings.scala:30]: How else we could configure java options? As this is private anymore

Comment (slandelle) [reply src/main/scala/io/gatling/sbt/settings/BaseSettings.scala:30]: We're going to revert. Thanks for reporting.

Comment (slandelle) [reply src/main/scala/io/gatling/sbt/settings/BaseSettings.scala:30]: We've released 4.1.1 with the revert.

**Pull request merged**